pithos/pithos.py

```diff
@@ -300,8 +299,8 @@ def init_core(self):
         self.equalizer = Gst.ElementFactory.make("equalizer-10bands", "equalizer-10bands")
         audioconvert = Gst.ElementFactory.make("audioconvert", "audioconvert")
         audioresample = Gst.ElementFactory.make("audioresample", "audioresample")
-        audioresample.set_property("quality", GstAudio.AUDIO_RESAMPLER_QUALITY_MAX)
-        audioresample.set_property("sinc-filter-mode", GstAudio.AudioResamplerFilterMode.FULL)
+        audioresample.set_property("quality", 10)
```

Comment:
At least define them as usable names elsewhere, and maybe leave a comment saying these constants are missing in X version of the gstreamer bindings.

Reply:
OK